scripts/background.js

```diff
@@ -42,14 +42,27 @@ setInterval(() => {
                     });
                 };
             } else if (!regex.settings.test(url) && !regex.shopping.test(url) && !audio.paused && !audio.src.includes(audioSource.health)) {
-                audio.pause();
+                fadeOut();
             };
         });
 }, 1000);
 
+function fadeOut() {
+    let volume = audio.volume;
```

Comment:
I suggest adding "initialVolume" to be able to set the volume to its previous value.

```js
const initialVolume = audio.volume;
const fadeOutInterval = setInterval(() => {
    if (audio.volume > 0) {
        audio.volume -= 0.01;
    } else {
        clearInterval(fadeOutInterval);
        audio.pause();
        audio.volume = initialVolume;
    }
}, 1);
```

I haven't changed these changes fully yet. Let me know if you find any issues.

Reply:
> Added it in.

Apparently not. It does not show up on GitHub.

Reply:
I see it on my commit list on my repo under commit bae4bf3. I was able see it fine. Try checking it again.

Reply:
Updated it anyways so it should now definitely appear for you.